shop/src/main/java/ro/msg/learning/shop/utils/mapper/ProductMapper.java

```diff
@@ -0,0 +1,48 @@
+package ro.msg.learning.shop.utils.mapper;
+
+import ro.msg.learning.shop.dto.ProductDto;
+import ro.msg.learning.shop.dto.ProductToSaveDto;
+import ro.msg.learning.shop.model.Product;
+import ro.msg.learning.shop.model.ProductCategory;
+import ro.msg.learning.shop.model.Supplier;
+
+public class ProductMapper {
+    public static ProductDto ProductToDto(Product product) {
+        if (product == null)
```

Comment:
Every time you have an if or else statement with only one line of logic be sure to also add the `{ }` because it might cause trouble when another developer works with it and doesn't notice that the brackets are missin

Reply:
really really useful piece of advice i have previously stumbled upon. thanks :D